tooling/gatsby-plugin/src/provider.js

```diff
@@ -0,0 +1,36 @@
+import * as React from "react"
+import {
+  ThemeProvider,
```

Comment:
https://chakra-ui.com/guides/integrations/with-gatsby#chakra-providers lists a `ChakraProvider`. Does that make a difference?

Reply:
Whew, that page doesn't do a good job of providing any context around `ChakraProvider`. 

`ChakraProvider` essentially handles all of the things that the `content` variable on line 20 is doing. It'd probably make more sense for `WrapRootElement` to accept [the provider's props](https://chakra-ui.com/docs/getting-started#chakraprovider-props) so they could be spread onto the provider and users can directly override them. 

I'm not sure if that would make sense for `theme` or not though. I suppose with shadowing, it would still make sense for that to be handled through `./theme` rather than passed directly through a prop, right?

Reply:
> I'm not sure if that would make sense for theme or not though. I suppose with shadowing, it would still make sense for that to be handled through ./theme rather than passed directly through a prop, right?

Yeah, shadowing is better here! With that you can hot reload the changes to the theme. If you would pass the theme via `gatsby-config` into the plugin options you'd need to restart the server every time 😬